src/helpers/helpers/plot.py

```diff
@@ -11,19 +11,20 @@ def group_box_plots(scores, groups, attr, group_names=None):
     Helper function for plotting group box curves. Assumes binary labels.
     """
     unique_groups = sorted(set(groups))
-    return go.Figure(
+    fig = go.Figure(
         data=[
             go.Box(x=groups[attr == a], y=scores[attr == a], name=a)
```

Comment:
think you need to swap `x` and `y` here, also you can just add `orientation="h"` as a keyword argument to `Box` rather than adding it below.

Reply:
Oops, that's right. I had this in my notebook version of the plot function then copied incorrectly into the script.